### ISSUE TITLE
fix this isssue please

### ISSUE THREAD

**Issue by fisforfaheem** [darioielardi/flutter_speed_dial#304]
The error you are encountering seems to be related to the compilation of your Flutter project for the web. There are a few issues reported in the log that need to be addressed:

"No such file or directory" error: The log indicates that it cannot find the Flutter directory. Make sure you have properly set up Flutter and that the path to the Flutter directory is correct.

"Woah! You appear to be trying to run flutter as root" warning: Running Flutter as root is not recommended. It's best to run Flutter without superuser privileges to avoid potential issues. Try running the command without superuser privileges.

"Error: The argument type 'BorderRadius?' can't be assigned to the parameter type 'BorderRadiusGeometry'" error: This error is related to incompatible types in the flutter_speed_dial package. Check if there is an updated version of the package available that resolves this issue. Alternatively, you can try removing or replacing the code that uses the flutter_speed_dial package to see if the error is resolved.

Other warnings and dependency issues: The log also mentions other warnings and overridden dependencies. It's recommended to review these warnings and resolve any conflicts or outdated dependencies. Update the dependencies mentioned in the log to their latest versions if possible.

To summarize, here are the steps you can take to address the errors:

Verify that the Flutter directory is correctly set up and accessible.
Run the Flutter commands without superuser privileges.
Check for an updated version of the flutter_speed_dial package and update it if available. Alternatively, remove or replace the code that uses the package.
Review the other warnings and dependency issues mentioned in the log and resolve them by updating the dependencies to their latest versions.
By addressing these issues, you should be able to resolve the compilation error and successfully build your Flutter project for the web.

**Comment by BenPoell** [comment]
Hi @fisforfaheem,
thanks for your issue. 
I'm not the author of this package but for me it seems that somehow some basic questions are not answered to be able to tackle your issue.

- what exactly are you trying to do? Do you build your project with `flutter build web` including the package?
- what is the exact output of the build/console?
- which flutter version are you using (best run `flutter doctor` and put the output here)

If you already have a clue how to solve this isse, you can fork this repository and make a PR - this makes it easier for the author to fix minor issues.
I made a PR, it should fix this issue:
https://github.com/darioielardi/flutter_speed_dial/pull/306

You can try it with my fork on your local machine and give feedback here:
```
dependency_overrides:
  flutter_speed_dial:
    git:
      url: https://github.com/BenPoell/flutter_speed_dial
```

Cheers, BP

**Comment by fisforfaheem** [comment]
Thanks man.

**Comment by BenPoell** [comment]
@prateekmedia this one should be fixed with #306. I guess you can reference and close it.
